packages/good-design/src/apps/goodid/components/SegmentationRow.tsx

```diff
@@ -33,21 +33,19 @@ const SegmentationRow = ({
     <VStack space={onCheck ? 2 : undefined}>
       <Title variant="subtitle-grey">{typeLabels[typeName]}</Title>
       {onCheck ? (
-        <Center>
-          <HStack space={4} width="100%" justifyContent={"flex-start"}>
-            <Checkbox
-              width="100%"
-              value={verifiedValue}
-              onChange={isChecked => onCheck(isChecked)}
-              colorScheme="info"
-              accessibilityLabel={`Dispute ${verifiedValue}`}
-            >
-              <Title variant="title-gdblue" textAlign="center" width="100%" minWidth={260}>
-                {verifiedValue}
-              </Title>
-            </Checkbox>
-          </HStack>
-        </Center>
+        <HStack space={4} width="95%" justifyContent="center">
```

Comment:
why 95%? usually paddings or margins should be used

Reply:
solved with padding